apps/accounts/urls.py

```diff
@@ -151,4 +154,24 @@
         TemplateView.as_view(template_name="provider_portal/before_starting.html"),
         name="before-starting",
     ),
+    path(
```

Comment:
These views we likely want to  scope to a specifc provider, so something like `/provider/<id>/verified-domains`